node_package/tests/jest.setup.js

```
@@ -62,3 +62,13 @@
   global.ReadableStreamDefaultReader = ReadableStreamDefaultReader;
 }
+
+global.console.log("All calls to console have been disabled in jest.setup.js")
+
+global.console = {
+  log: jest.fn(),
+  error: jest.fn(),
+  warn: jest.fn(),
+  info: jest.fn(),
+  debug: jest.fn(),
+}
```

ESLint

[error] Insert `;`

(prettier/prettier)